storage/backup/backup-cli/src/backup_types/transaction/restore.rs

```diff
@@ -259,9 +263,11 @@ impl TransactionRestoreBatchController {
         }
 
         let mut loaded_chunk_stream = self.loaded_chunk_stream();
-        let first_version = self
-            .confirm_or_save_frozen_subtrees(&mut loaded_chunk_stream)
-            .await?;
+        // first_version determines the start version for saving txns
+        let first_version = self.first_version.unwrap_or(
+            self.confirm_or_save_frozen_subtrees(&mut loaded_chunk_stream)
```

Comment:
If the first_version is None, we are still gone restore txn from the first version of the chunk.  Ex, restore DB to latest version while saving txn ws from all loaded chunks.

Reply:
I see, I misread, I thought you always recover the frozen subtrees according to the chunk.first_version, but infact you skip it when there's self.first_versino.

But when first version is not None, we should also `confirm_or_save_frozen_subtrees`? otherwise the txn accumulator is not valid.

Reply:
If the first version is not None, we don't want to do anything since this is in the 2nd phase restoring after the 2nd snapshot when the initial TXN ACC was already saved.